src/app/content/components/Toolbar.tsx

```diff
@@ -0,0 +1,152 @@
+import React, { SFC } from 'react';
+import styled from 'styled-components';
+import {ListOl} from 'styled-icons/fa-solid/ListOl';
+import {Search} from 'styled-icons/fa-solid/Search';
+import {Print} from 'styled-icons/fa-solid/Print';
+import theme from '../../theme';
+import Typography from '../../components/Typography';
+
+const iconStyles = {
+  mobile: {
+    height: '1.6rem',
+    width: '1.6rem',
+  },
+}
+
+const ListIcon = styled(ListOl)`
+  height: ${iconStyles.mobile.height};
+  width: ${iconStyles.mobile.width};
+  color: ${theme.color.secondary.darkBlue.base};
+  margin-right: 0.7rem;
+
+  :hover {
+    color: ${theme.color.secondary.darkBlue.foreground};
+  }
+
+`;
+
+const SearchIcon = styled(Search)`
+  height: ${iconStyles.mobile.height};
+  width: ${iconStyles.mobile.width};
+  color: ${theme.color.secondary.darkGray.base};
+  margin-right: 0.7rem;
+`;
+
+const PrintIcon = styled(Print)`
+  height: ${iconStyles.mobile.height};
+  width: ${iconStyles.mobile.width};
+  color: ${theme.color.secondary.darkGray.base};
+  margin-right: 0.7rem;
+
+`;
+
+const TopBar = styled.div`
+  height: 5rem;
+  max-width: 117rem;
+  margin: 0 auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  @media (max-width: 48em) {
+    height: 4rem;
+  }
+  
+`;
+
+const ToCButton = styled.h3`
+  ${Typography.bodyCopyRegularStyle};
+  color: ${theme.color.secondary.darkBlue.base};
+  cursor: pointer;
+  margin: 0;
+
+  :hover {
+    color: ${theme.color.secondary.darkBlue.foreground};
+  }
+
+`;
+
+const SearchInputWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  margin-right: 4rem;
+  border-bottom: solid 0.1rem ${theme.textColors.default};
+
+  @media (max-width: 48em) {
+    border: none;
+    margin-right: 1rem;
+  }
+`;
+
+const SearchInput = styled.input`
+  ${Typography.bodyCopyRegularStyle};
+  color: ${theme.textColors.default};
+  border: none;
+  height: 2.5rem;
```

Comment:
you might not need this, since you get the line height from the bodyCopyRegularStyle